xsd/netex-nl.xsd

```diff
@@ -238,13 +243,6 @@
 							</xsd:sequence>
 						</xsd:complexType>
 					</xsd:element>
-					<xsd:element name="organisations">
-						<xsd:complexType>
-							<xsd:sequence>
-								<xsd:element name="Operator" type="operator" maxOccurs="unbounded"/>
-								<xsd:element name="Organisation" type="organisation" minOccurs="0" maxOccurs="unbounded"/>
```

Comment:
Ik denk niet dat dit weg moet, maar dat hier _eigenlijk_ Authority bij zou moeten. Authority staat nu alleen in het GeneralFrame.